streams/streams-scala/src/test/scala/org/apache/kafka/streams/scala/kstream/KStreamTest.scala

```diff
@@ -448,27 +478,27 @@ class KStreamTest extends TestDriver {
     assertEquals("my-name", joinNode.name())
   }
 
-  @nowarn
-  @Test
-  def testSettingNameOnTransform(): Unit = {
-    class TestTransformer extends Transformer[String, String, KeyValue[String, String]] {
-      override def init(context: ProcessorContext): Unit = {}
-
-      override def transform(key: String, value: String): KeyValue[String, String] =
-        new KeyValue(s"$key-transformed", s"$value-transformed")
-
-      override def close(): Unit = {}
-    }
-    val builder = new StreamsBuilder()
-    val sourceTopic = "source"
-    val sinkTopic = "sink"
-
-    val stream = builder.stream[String, String](sourceTopic)
-    stream
-      .transform(() => new TestTransformer, Named.as("my-name"))
-      .to(sinkTopic)
-
-    val transformNode = builder.build().describe().subtopologies().asScala.head.nodes().asScala.toList(1)
-    assertEquals("my-name", transformNode.name())
-  }
+//  @nowarn
+//  @Test
+//  def testSettingNameOnTransform(): Unit = {
+//    class TestTransformer extends Transformer[String, String, KeyValue[String, String]] {
+//      override def init(context: ProcessorContext): Unit = {}
+//
+//      override def transform(key: String, value: String): KeyValue[String, String] =
+//        new KeyValue(s"$key-transformed", s"$value-transformed")
+//
+//      override def close(): Unit = {}
+//    }
+//    val builder = new StreamsBuilder()
+//    val sourceTopic = "source"
+//    val sinkTopic = "sink"
+//
+//    val stream = builder.stream[String, String](sourceTopic)
+//    stream
+//      .transform(() => new TestTransformer, Named.as("my-name"))
+//      .to(sinkTopic)
+//
+//    val transformNode = builder.build().describe().subtopologies().asScala.head.nodes().asScala.toList(1)
+//    assertEquals("my-name", transformNode.name())
+//  }
```

Comment:
Should we rewrite this to `testSettingNameOnProcess`?

Reply:
Oh, sorry! I have missed it up. Of course, I will rewrite it!